Sources/ApolloCodegenLib/Templates/RenderingHelpers/OperationTemplateRenderer.swift

```diff
@@ -24,7 +24,7 @@ extension OperationTemplateRenderer {
     _ variables: [CompilationResult.VariableDefinition]
   ) -> TemplateString {
     """
-    \(variables.map { "public var \($0.name.asInputParameterName): \($0.type.rendered(as: .inputValue, config: config.config))"}, separator: "\n")
+    \(variables.map { "public var \($0.name.asFieldAccessorPropertyName): \($0.type.rendered(as: .inputValue, config: config.config))"}, separator: "\n")
```

Comment:
If I read this right we could have a situation where we're creating the property name here with `asFieldAccessorPropertyName` but referring to it with a name generated by `asInputParameterName` up on [line 17](https://github.com/apollographql/apollo-ios/pull/2996/files#diff-905e0defd35e8857aa7177892f45520af989a81895465ffb4f447e53e36dfed2R17). I think it works for now because they actually seem to escape the same list of strings but having them be separate is leaving room for error. Is it worth reducing this into a single function and corresponding list-to-be-escaped?

Reply:
Oh good catch, I was using `asInputParameterName` there because it was for the input parameter of the `init` but didn't catch that it would be accessing the generated `var` name. Right now `asInputParameterName` and `asFieldAccessorPropertyName` are both functioning exactly the same, so we could definitely combine them into one function. I can't think of a need to have input parameters for functions formatted differently than variables but if that case did come up we could always break this back out in the future.

Reply:
> I can't think of a need to have input parameters for functions formatted differently than variables but if that case did come up we could always break this back out in the future.

Me neither, I think it'd be a good decision to combine them.